dcp_prototype/backend/artifact_manager/schemas/v0.0.0.json

```diff
@@ -0,0 +1,408 @@
+{
+   "definitions":{
+
+   },
+   "version":"v0.1.0",
+   "$schema":"http://json-schema.org/draft-07/schema#",
+   "$id":"http://data.humancellatlas.org.com/artifact.json",
+   "type":"object",
+   "title":"The Artifact Schema",
+   "required":[
+      "projects",
```

Comment:
Two more things that would be useful: 

scheme_version:  equal to the version of the schema.  It would be useful to have this in the file so that the code that processes it can handle more than one version.   

schema_name: equal to something like "DCP Artifact Schema", so that the json file is more self describing.  

Other useful metadata:
   version (or maybe date): version (or date) of the current contents of the file.  Useful to know something about the when the data was assembled.
  title : some information about what data is actually here.  E.g. "Demo Dataset", "Production Dataset", "Test Dataset".

Reply:
Added a `meta` section with this information.